src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKAnnotationArgumentCollection.java

```diff
@@ -29,8 +29,12 @@
     public abstract List<String> getUserDisabledAnnotationNames();
 
     /**
-     * Returns {@code true} if all default annotations are disabled; {@code false} otherwise.
+     * Return {@code true} if the user has requested that all annotations be enabled; {@code false} otherwise.
```

Comment:
Docs are wrong here, and basically the opposite of what they should be: `Returns {@code true} if all tool-default annotations are disabled; {@code false} otherwise.`